mail-filter/imapfilter/imapfilter-2.8.2.ebuild

```diff
@@ -13,7 +13,7 @@ SRC_URI="https://github.com/lefcha/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="MIT"
 SLOT="0"
-KEYWORDS="amd64 ppc x86"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
```

Comment:
revert this, this isn't how we add arches.

Please read [devmanual](https://devmanual.gentoo.org/keywording/index.html).